warehouse/types.go

```diff
@@ -1,13 +1,16 @@
 package warehouse
 
 import (
+	"errors"
 	"sync"
 	"time"
 
 	"github.com/rudderlabs/rudder-server/utils/misc"
 	"github.com/rudderlabs/rudder-server/warehouse/internal/model"
 )
 
+var ContextError = errors.New("[error] pq: canceling statement due to user request")
```

Comment:
minor: Can we move this to the errors package?

```suggestion
var ErrCancellingStatement = errors.New("[error] pq: canceling statement due to user request")
```